README.md

```diff
@@ -2,26 +2,35 @@
   <a href="https://clerk.dev?utm_source=github&utm_medium=clerk_docs" target="_blank" rel="noopener noreferrer">
     <picture>
       <source media="(prefers-color-scheme: dark)" srcset="https://images.clerk.dev/static/logo-dark-mode-400x400.png">
-      <img src="https://images.clerk.dev/static/logo-light-mode-400x400.png" height="64">
+      <img src="https://images.clerk.dev/static/logo-light-mode-400x400.png" height="120">
```

Comment:
It looks like most of our other READMEs use the same size logo, should we keep it the smaller size for consistency?

ref:
* https://github.com/clerkinc/javascript/blob/main/README.md